src/cli/cli.hpp

```diff
@@ -194,6 +205,25 @@ class Interpreter
      */
     void OutputLine(const char *aFormat, ...);
 
+    /**
+     * This method delivers formatted output (to which it prepends a given number indentation space chars and appends
+     * newline `\r\n`) to the client.
+     *
+     * @param[in]  aIndentSize   Number of indentation space chars to prepend to the string.
+     * @param[in]  aFormat       A pointer to the format string.
+     * @param[in]  ...           A variable list of arguments to format.
+     *
+     */
+    void OutputLine(uint16_t aIndentSize, const char *aFormat, ...);
+
+    /**
+     * This method writes a given number of space chars to the CLI console.
+     *
+     * @param[in] aCount  Number of space chars to output.
+     *
+     */
+    void OutputSpaces(uint16_t aCount);
```

Comment:
Given this is not code that not necessary to consider tab width difference, is it possible to use tab for indentation?

Reply:
If you are suggesting to use `'\t'` instead of `' '` (space) for indentation, I won't recommend it as the actual tab width would be platform dependent and I think it would be better to have a consistent output for CLI commands on all platforms - Also we used to use `\t` in some of the logs and it caused problems on some internal products/platforms.

Existing code already seems to uses different indentation sizes of 4, 8, **10**, 12 in different places (e.g. see `OutputRoute()`).